.travis.yml

```diff
@@ -1,6 +1,8 @@
+dist: trusty
+sudo: false
```

Comment:
Yeah, I think it's a good idea in general to prepare for the upgrade, especially we also run this version.